src/frontend/hooks/useProjectInviteListener.ts

```diff
@@ -1,14 +1,25 @@
-import * as React from "react";
-import { useNavigation } from "react-navigation-hooks";
-import ProjectInviteContext from "../context/ProjectInviteContext";
-
-export default function useProjectInviteListener() {
-  const navigation = useNavigation();
-  const { invite } = React.useContext(ProjectInviteContext);
-
-  React.useEffect(() => {
-    if (invite) {
-      navigation.navigate("ProjectInviteModal");
-    }
-  }, [invite, navigation]);
+import { useEffect, useRef } from "react";
+
+// Gregor TODO: Add method to API for (un)subscribing to project invites
+const subscribeToProjectInvites = (callback: (inviteKey: string) => void) => {
+  const intervalId = setInterval(() => callback("abc123"), 3000);
+  return () => clearInterval(intervalId);
+};
```

Comment:
Placeholder for when the API supports the subscribe to project invites functionality